### PR TITLE
HU translation added nitter addon THX urbalazs

### PR DESCRIPTION
**Pull request by tobiasd**

### CODE REVIEW

**Comment by tobiasd** [comment]
wait a moment...

File: nitter/lang/de/strings.php [hunk 1]
```diff
@@ -8,3 +8,4 @@ function string_plural_select_de($n){
 ;
 $a->strings["Nitter server"] = "Nitter Server";
 $a->strings["Save Settings"] = "Einstellungen Speichern";
+$a->strings["In an attempt to protect your privacy, links to Twitter in this posting were replaced by links to the Nitter instance at %s"] = "Um deine Privatsphäre zu schützen, wurden in diesem Beitrag Links nach ";
```
Comment (annando) [inline nitter/lang/de/strings.php:11]: This looks incomplete

Comment (tobiasd) [reply nitter/lang/de/strings.php:11]: You are right, all the split lines need to be removed :-/

**Comment by tobiasd** [comment]
Closing this, the problem needs to be fixed in the po2php command.

**Pull request closed without merging**